CHANGES.md

```diff
@@ -1,6 +1,10 @@
 # Unreleased
 
-- [NEW] Add `CloudantClient.metaInformation()`.
+- [NEW] Add the following methods:
```

Comment:
Maybe add a piece that the methods are part of `_scheduler` endpoint? e.g. `Added the following _scheduler endpoint methods`

Reply:
Yes, I think this should be something more like:
```
- [NEW] Added `CloudantClient.metaInformation()` to get more meta information from the welcome page.
- [NEW] Added methods for interacting with the replicator's `_scheduler` endpoint: ...
```